docs/src/modules/concepts/pages/saga-patterns.adoc

```diff
@@ -1,56 +1,60 @@
 = Saga patterns
 
-include::ROOT:partial$include.adoc[]
+Saga patterns help manage long-running business processes in distributed systems by dividing them into a series of transactions. Each transaction either completes successfully or triggers compensating actions if something goes wrong.
```

Comment:
Feel free to ignore: should we use something like `atomic action` instead of the more specific `transaction` ?